src/commands/logstream/enableFileLogging.ts

```diff
@@ -24,14 +25,14 @@ export async function enableFileLogging(context: IEnableFileLoggingContext, node
         node = <SiteTreeItem>node.parent;
     }
 
-    const siteNode: SiteTreeItem = node;
+    const siteNode: SiteTreeItem | TrialAppTreeItem = node;
 
     const isEnabled = await vscode.window.withProgress({ location: vscode.ProgressLocation.Notification }, async p => {
         p.report({ message: 'Checking container diagnostics settings...' });
         return await siteNode.isHttpLogsEnabled();
     });
 
-    if (!isEnabled) {
+    if (!isEnabled && siteNode instanceof SiteTreeItem && node instanceof SiteTreeItem) {
```

Comment:
Why do you have to check both `siteNode` and `node`? I would think you should only be using `siteNode` after line 28